### DISCUSSION TITLE
Stand Up Friday April 21th

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [Bugs-io/TC3001B#64] [Main Stand Ups]
```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished signup logic
 - Setup PostgreSQL on my laptop

**What are your priorities for the next 24 hours?**:
 - Finish login logic
 - Start working on analytics service with @Jacobgtz024 

**Blockers**:
 - None

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished web activity 
 
**What are your priorities for the next 24 hours?**:
 - Help @CarlosCRG19  finish login logic on backend 
 
**Blockers**:
 - None

**Shoutouts**:
 - @Jacobgtz024  for his interview with Oracle, best of luck!

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Implemented fields verifications on signups
 - Added password strength meter

**What are your priorities for the next 24 hours?**:
- Finished login and signup
**Blockers**:
 
**Shoutouts**:
 - @BrandonMagana for dockerizing the web activity
